### PR TITLE
Add Robosense LiDAR rslidar-16 and rslidar-32

### PR DESCRIPTION
**Pull request by Tony-HIT**

### CODE REVIEW

**Comment by lianglia-apollo** [comment]
Thanks for contributing to Apollo! Please sign the CLA to merge the code.

File: modules/drivers/rslidar/rslidar_pointcloud/include/rslidar_pointcloud/rslidarParser.h~ [hunk 1]
```diff
@@ -0,0 +1,259 @@
+/******************************************************************************
+ * Copyright 2017 The Apollo Authors. All Rights Reserved.
```
Comment (mickeyouyou) [inline modules/drivers/rslidar/rslidar_pointcloud/include/rslidar_pointcloud/rslidarParser.h~]: `rslidarParser.h~` please remove this unused files

File: .gitignore [hunk 2]
```diff
@@ -88,3 +88,6 @@ modules/calibration/data/mkz056
 
 # Data bag
 docs/demo_guide/*.bag
+
```
Comment (lianglia-apollo) [inline .gitignore]: please delete the comments.

File: modules/drivers/rslidar/rslidar/scripts/start_velodyne.sh [hunk 3]
```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+###############################################################################
+# Copyright 2017 The Apollo Authors. All Rights Reserved.
```
Comment (lianglia-apollo) [inline modules/drivers/rslidar/rslidar/scripts/start_velodyne.sh]: 2017 --> 2018

File: modules/drivers/rslidar/rslidar_driver/include/rslidar_driver/input.h [hunk 4]
```diff
@@ -0,0 +1,64 @@
+/******************************************************************************
+ * Copyright 2017 The Apollo Authors. All Rights Reserved.
```
Comment (lianglia-apollo) [inline modules/drivers/rslidar/rslidar_driver/include/rslidar_driver/input.h]: 2017 --> 2018

File: modules/drivers/rslidar/rslidar_driver/include/rslidar_driver/socket_input.h [hunk 5]
```diff
@@ -0,0 +1,55 @@
+/******************************************************************************
+ * Copyright 2017 The Apollo Authors. All Rights Reserved.
```
Comment (lianglia-apollo) [inline modules/drivers/rslidar/rslidar_driver/include/rslidar_driver/socket_input.h]: 2018

File: modules/drivers/rslidar/rslidar_pointcloud/data/lidar1/curves.csv [hunk 6]
```diff
@@ -307,7 +307,7 @@
 5.1331,4.2384,4.6339,3.9479,3.6514,5.667,3.9687,3.6054,3.6245,4.3939,6.0772,2.8687,3.3689,5.2192,4.2651,7.2788
 5.1314,4.2258,4.6339,3.947,3.6529,5.6541,3.9689,3.6054,3.6278,4.3863,6.0737,2.8704,3.3616,5.2157,4.2625,7.2421
 5.1298,4.2136,4.6339,3.9461,3.6544,5.6414,3.9692,3.6054,3.6311,4.3787,6.0701,2.8721,3.3542,5.2124,4.26,7.2057
-5.1282,4.2017,4.6339,3.9452,3.656,5.6288,3.9694,3.6052,3.6345,4.3711,6.0664,2.8739,3.3467,5.2091,4.2575,7.1698
```
Comment (mickeyouyou) [inline modules/drivers/rslidar/rslidar_pointcloud/data/lidar1/curves.csv]: Are you sure to replace the value of curves here ?

File: modules/drivers/rslidar/rslidar_pointcloud/data/lidar1/curves.csv [hunk 7]
```diff
@@ -370,7 +370,7 @@
 5.0674,3.9551,4.6865,3.9281,3.783,5.0467,3.9821,3.5421,3.7674,3.9582,5.7088,2.9829,3.0135,5.1943,4.1577,5.6793
 5.067,3.9555,4.6882,3.9284,3.7856,5.0398,3.9817,3.5421,3.7672,3.9527,5.7017,2.9829,3.0125,5.1967,4.1573,5.6637
 5.0666,3.9559,4.6899,3.9287,3.7883,5.0329,3.9813,3.5423,3.7669,3.9473,5.6945,2.9829,3.0116,5.1992,4.157,5.6484
-5.0662,3.9563,4.6916,3.9291,3.791,5.0261,3.9809,3.5426,3.7665,3.942,5.6873,2.9827,3.0109,5.2017,4.1568,5.6332
```
Comment (mickeyouyou) [inline modules/drivers/rslidar/rslidar_pointcloud/data/lidar1/curves.csv]: Are you sure to replace the value in this curves file ?

File: modules/drivers/rslidar/rslidar_pointcloud/data/lidar1/curves.csv (hunk 6 above)
Comment (Tony-HIT) [reply modules/drivers/rslidar/rslidar_pointcloud/data/lidar1/curves.csv]: Yes. It does not influence the lidar. Each lidar has its own calibration file, so this file will be replaced when users bring up their own lidar.

File: modules/drivers/rslidar/rslidar_pointcloud/data/lidar1/curves.csv (hunk 7 above)
Comment (Tony-HIT) [reply modules/drivers/rslidar/rslidar_pointcloud/data/lidar1/curves.csv]: Yes. It does not influence the lidar. Each lidar has its own calibration file, so this file will be replaced when users bring up their own lidar.

**Pull request merged**